_filters/sum.md

```diff
@@ -0,0 +1,40 @@
+---
+title: sum
+description: Liquid filter that sums all items in an array.
+---
+
+Sums all items in an array.
+
+<p class="code-label">Input</p>
+```liquid
+{%- raw -%}
+<!-- if array = [1,2,3] -->
+{{ array | sum }}
+{% endraw %}
+```
+
+<p class="code-label">Output</p>
+```text
+6
+```
+
+ If a string is passed as an argument, it sums the property values.
+
+<p class="code-label">Input</p>
+```liquid
+{%- raw -%}
+<!-- if array = [
+  {quantity: 1, weight: 2},
+  {quantity: 2},
+  {quantity: 3}
+]
```

Comment:
Since liquid does not have object literals, it might be beneficial to use an intention-revealing array of objects (e.g., `collection.products`) like they do in the `map` documentation (`"In this example, assume the object site.pages contains all the metadata for a website..."`)